buildSrc/build.gradle.kts

```diff
@@ -28,6 +28,8 @@ repositories {
 dependencies {
     implementation("com.autonomousapps:dependency-analysis-gradle-plugin:1.20.0")
     implementation("com.diffplug.spotless:spotless-plugin-gradle:6.15.0")
+    implementation("com.google.protobuf:protobuf-gradle-plugin:0.9.3")
+    implementation("com.hedera.pbj:pbj-compiler:0.6.0")
```

Comment:
Is there a reason these version are not in catalog?
PBJ in particular will change quite often and need to be managed by people and will need to match for compiler and runtime

Reply:
The reason was consistency. But I did not think of the `pbj-compiler` to be "special" compared to the other plugins. I reverted this.